packages/react-native-editor/CHANGELOG.md

```diff
@@ -11,13 +11,20 @@ For each user feature we should also add a importance categorization label  to i
 
 ## Unreleased
 
+## 1.50.1
+
+-   [x] Truncate rangecell screenreader decimals] [#30678]
```

Comment:
@guarani is this a new format we are using for release notes entries? Let me know because I am trying to figure out if we need to put any of these updates in the release notes of the main apps.

Reply:
Thanks for spotting! It's a mistake. I'll fix this next week since like you said offline it's not strictly needed for the release.